.goreleaser.yml

```diff
@@ -6,7 +6,6 @@ before:
 builds:
   -
     main: ./cmd/synse.go
-    binary: synse
```

Comment:
I'm a bit hazy on this change, probably worth double checking.

Reply:
@MatthewHink The yaml schema of goreleaser changed, so if this is not changed, you get errors like:

```
   • loaded config file        config={octool [] {vapor-ware/octool   false false auto {{ .Version }} [] []    } [] [{octool {vapor-ware homebrew-formula  } {vio-bot marco+viogh@vapor.io}       [] system "#{bin}/octool -h"

 [] Simple tool to test/debug/diagnose connections to OpenConfig servers https://github.com/vapor-ware/octool       [] }] [] [] { {   }  { }      []  [] []} [{ [linux darwin] [amd64] [] [] [] []   [-w -s] [] []  {[] []} [CGO_ENABLED=0]  [] []  false  false  }] [] [] [] {{{ .Tag }}-next} {checksums.txt  [] false []} [{ [] linux amd64   [vaporio/octool:{{ .Tag }} vaporio/octool:{{ .Major }} vaporio/octool:latest]  [] [] [] false }] [] [] [] [] [] {{[^docs: ^test:]} asc false  []}  [] [] {  } {[go mod download rm -rf dist]} {  false } {false [] } { {false } {false     } {false     } {false    } {false    } {false  0   []   false} {false       } {false } {false  0}} [] { [] [] [] [] [] []   [] [] []  {[] []} []  [] []  false  false  } {   false} {  false} {  false}}

   ⨯ release failed after 0.00s error=yaml: unmarshal errors:

  line 27: field binaries not found in type config.Docker

script returned exit code 1
```

Also mentioned here: https://goreleaser.com/deprecations/#dockerbinaries

Reply:
Woops, `binaries` was removed, not `binary`.  Adding `binary` back in.